cmds/exp/netbootxyz/netbootxyz.go

```diff
@@ -27,6 +39,7 @@ var (
 	verbose = flag.Bool("v", false, "Verbose output")
 	noLoad  = flag.Bool("no-load", false, "Get DHCP response, print chosen boot configuration, but do not download + exec it")
 	noExec  = flag.Bool("no-exec", false, "Download boot configuration, but do not exec it")
+	noDhcp  = flag.Bool("no-dhcp", false, "Don't try to get an IP address via DHCP")
```

Comment:
```suggestion
	noDHCP  = flag.Bool("no-dhcp", false, "Don't try to get an IP address via DHCP")
```

Uppercase acronyms is naming convention in Go